glide.lock

```diff
@@ -0,0 +1,60 @@
+hash: 24dd85605b7c94a4ecc2c7fd16496787124c613e5fe272ec102575af738b01c0
```

Comment:
We want to keep the lockfile so there is no surprise when building/testing. Different peoples copies of atc will be slightly different depending on when the project was downloaded and glide install ran.

This has been discussed pretty heavily in the ruby and rails community regarding Gemfile.lock.